dbt_subprojects/solana/models/solana_utils/alt_solana_utils_token_2022_account_raw_data.sql

```diff
@@ -0,0 +1,131 @@
+{{
+  config(
+    schema='solana_utils',
+    alias='alt_token_2022_account_raw_data',
+    materialized='incremental',
+    file_format='delta',
+    partition_by=['token_account_prefix'],
+    incremental_strategy='merge',
+    incremental_predicates=[incremental_predicate('DBT_INTERNAL_DEST.event_time')],
+    unique_key=['token_account', 'instruction_uniq_id']
+  )
+}}
+
+--Init events contain mint address, and owner address
+--Init v1
+SELECT
+  account_accountToInitialize AS token_account,
+  account_owner,
+  account_mint,
+  call_block_time AS event_time,
+  DATE_TRUNC('month', call_block_time) AS block_month,
+  SUBSTRING(account_accountToInitialize, 1, 2) AS token_account_prefix,
+  'init' AS event_type,
+  -- constructing an artificial instruction_uniq_id to order instructions using one string column
+  -- lpads are chosen carefully to be lexicographically sortable
+  -- we can sort the instructions by this column to get the correct order
+  CONCAT(
+      lpad(cast(call_block_slot as varchar), 12, '0'), '-', 
+      lpad(cast(call_tx_index as varchar), 6, '0'), '-', 
+      lpad(cast(coalesce(call_outer_instruction_index, 0) as varchar), 4, '0'), '-', 
+      lpad(cast(coalesce(call_inner_instruction_index, 0) as varchar), 4, '0')
+    ) AS instruction_uniq_id
```

Comment:
can we possibly use this made up key or should we go back to the actual values?

what are performance implications?